### ISSUE TITLE
Helm: Error: Failed to download "chart/helm3/sops-operator"

### ISSUE THREAD

**Issue by dsyorkd** [isindir/sops-secrets-operator#116]
Following instructions from readme: 

```
helm upgrade --install sops chart/helm3/sops-secrets-operator/  -n sops -f sops-values.yaml --debug                                                                                                                                  
history.go:56: [debug] getting history for release sops
Release "sops" does not exist. Installing it now.
install.go:178: [debug] Original chart version: ""
Error: repo chart not found
helm.go:84: [debug] repo chart not found
helm.sh/helm/v3/pkg/downloader.pickChartRepositoryConfigByName
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:368
helm.sh/helm/v3/pkg/downloader.(*ChartDownloader).ResolveChartVersion
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:253
helm.sh/helm/v3/pkg/downloader.(*ChartDownloader).DownloadTo
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:90
helm.sh/helm/v3/pkg/action.(*ChartPathOptions).LocateChart
	helm.sh/helm/v3/pkg/action/install.go:753
main.runInstall
	helm.sh/helm/v3/cmd/helm/install.go:190
main.newUpgradeCmd.func2
	helm.sh/helm/v3/cmd/helm/upgrade.go:121
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.4.0/command.go:856
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.4.0/command.go:974
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.4.0/command.go:902
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
	runtime/proc.go:255
runtime.goexit
	runtime/asm_arm64.s:1133
```

Using explicit version: 
```
helm upgrade --install sops chart/helm3/sops-secrets-operator  -n sops -f sops-values.yaml --debug --version 0.12.1
history.go:56: [debug] getting history for release sops
Release "sops" does not exist. Installing it now.
install.go:178: [debug] Original chart version: "0.12.1"
Error: repo chart not found
helm.go:84: [debug] repo chart not found
helm.sh/helm/v3/pkg/downloader.pickChartRepositoryConfigByName
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:368
helm.sh/helm/v3/pkg/downloader.(*ChartDownloader).ResolveChartVersion
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:253
helm.sh/helm/v3/pkg/downloader.(*ChartDownloader).DownloadTo
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:90
helm.sh/helm/v3/pkg/action.(*ChartPathOptions).LocateChart
	helm.sh/helm/v3/pkg/action/install.go:753
main.runInstall
	helm.sh/helm/v3/cmd/helm/install.go:190
main.newUpgradeCmd.func2
	helm.sh/helm/v3/cmd/helm/upgrade.go:121
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.4.0/command.go:856
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.4.0/command.go:974
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.4.0/command.go:902
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
	runtime/proc.go:255
runtime.goexit
	runtime/asm_arm64.s:1133
```

**Comment by dsyorkd** [comment]
without /helm3/ : 
```
helm upgrade --install sops chart/sops-secrets-operator/ --namespace sops -f sops-values.yaml --debug 
history.go:56: [debug] getting history for release sops
Release "sops" does not exist. Installing it now.
install.go:178: [debug] Original chart version: ""
Error: repo chart not found
helm.go:84: [debug] repo chart not found
helm.sh/helm/v3/pkg/downloader.pickChartRepositoryConfigByName
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:368
helm.sh/helm/v3/pkg/downloader.(*ChartDownloader).ResolveChartVersion
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:253
helm.sh/helm/v3/pkg/downloader.(*ChartDownloader).DownloadTo
	helm.sh/helm/v3/pkg/downloader/chart_downloader.go:90
helm.sh/helm/v3/pkg/action.(*ChartPathOptions).LocateChart
	helm.sh/helm/v3/pkg/action/install.go:753
main.runInstall
	helm.sh/helm/v3/cmd/helm/install.go:190
main.newUpgradeCmd.func2
	helm.sh/helm/v3/cmd/helm/upgrade.go:121
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.4.0/command.go:856
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.4.0/command.go:974
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.4.0/command.go:902
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
	runtime/proc.go:255
runtime.goexit
	runtime/asm_arm64.s:1133
```

**Comment by isindir** [comment]
@dsyorkd , fix #121 changes readme to describe how to use helm repository instead of local directory from cloned repository. Appologies for late response, please try and let me know if that fixes your issue. 

As a side note - this method is supposed to be used with `helm` version 3, could you please report version of `helm` which you are using ?

Tnank you for reporting.
